### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by tcyrus**
* Replaced usage of `var` with `let` and `const` as necessary
* Handle `parseInt` failure case (`NaN`)

### CODE REVIEW

File: website/app.js [hunk 1]
```diff
@@ -56,6 +55,9 @@ app.all('/', (req, rsp, next) => {
 	let visit_count = 1;
 	if ('visit_count' in req.cookies) {
 		visit_count = parseInt(req.cookies.visit_count);
+		if (isNaN(visit_count)) {
```
Comment (dphiffer) [inline website/app.js]: this is great, I always forget about the `isNaN` function.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#The_necessity_of_an_isNaN_function

File: website/app.js [hunk 2]
```diff
@@ -22,8 +22,8 @@ function get_redirect_url() {
 		'2019-12-10': '14'
 	};
 
-	var current_url = base_url;
-	for (let class_date in classes) {
+	let current_url = base_url;
+	for (const class_date in classes) {
```
Comment (dphiffer) [inline website/app.js]: This `const` looked wrong at first but then I tested it out, and it works, so I may need to read up more on ES6 and iterator vars.

**Comment by dphiffer** [comment]
Testing this out I discovered a redundant trailing slash issue:
```
$ curl localhost:8001
Found. Redirecting to https://github.com/dphiffer/hacktivism//blob/master/classes/01.md
```
Fixed here: https://github.com/dphiffer/hacktivism/commit/808c5fc735b6d25760859a8f1632b544a3711c5f

File: website/app.js (hunk 2 above)
Comment (tcyrus) [reply website/app.js]: `const` just means that you can't modify the variable `class_date` inside the `for ... in` loop

**Comment by tcyrus** [comment]
> ```
> Found. Redirecting to https://github.com/dphiffer/hacktivism//blob/master/classes/01.md
> ```

I don't think that's related to the changes I made.

**Comment by dphiffer** [comment]
Nope, this is my bug, I just found it while checking your PR. Thanks for surfacing it :)

**Pull request merged**